### DISCUSSION TITLE
Git commit ID follows assembly version in doc output

### DISCUSSION THREAD

**Discussion by kaosborn** [EWSoftware/SHFB#1039] [Q&A]
After upgrading my `.shfbproj` projects to latest everything, there is a change in the assembly version reported in my doc output but only when the project is under version control.

![shfb-issue](https://github.com/EWSoftware/SHFB/assets/27261136/9e9bdc35-5238-4c45-90fa-d7bab2b74f62)

Here is what it looked like before upgrading project:

https://kaosborn.github.io/help/KaosCombinatorics/html/bbaadc50-10fa-dfcf-a4db-8532baaef11e.htm

The assembly version now includes the 40 hex digits of a Git commit ID. Is there anything I can do to get the same output regardless of whether or not the project is under version control?

**Comment by EWSoftware** [comment]
It's always pulled from the `AssemblyInformationalVersion` attribute if present or the `AssemblyVersion` attribute if not , that hasn't changed.  You've likely got a build task that's modifying the assembly version based on the commit ID or adding the informational attribute which is then taking precedence.  I can change the handling so that the plus is included as a separator.  Setting the max version parts to 4 would then prevent it from including the commit ID.  Setting it to 3 right now would work around it by trimming off the revision and everything after it.
